docs/src/main/sphinx/ext/download.py

```diff
@@ -22,10 +22,65 @@
     'server': ('trino-server', 'tar.gz', None),
     'cli': ('trino-cli', 'jar', 'executable'),
     'jdbc': ('trino-jdbc', 'jar', None),
-
+    #plugins
+    'ranger':  ('trino-ranger', 'zip', None),
```

Comment:
sort

Reply:
Ok .. maybe we should follow up with changing the plugin folder name for the ranger plugin from apache-ranger to ranger as well ..